src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String startTime;
+    protected String endTime;
+    public Event(String task, String startTime, String endTime) {
+        super(task, 'E');
+        this.startTime = startTime;
+        this.endTime = endTime;
+    }
+    @Override
+    public String toString() {
```

Comment:
I deliberately chose to overrise the toString method so that in the even that there is a need to print the tasks/event/deadline using the System.out.print, the toString method will be invoked

Reply:
Ok, that makes sense. So it accounts for the cases when you print those objects (toDo/Event/Deadline) directly, so it's more comprehensive in a way?